### ISSUE TITLE
Request the Node::sendfile() routine support multiple files.

### ISSUE THREAD

**Issue by djbusby** [chrome-php/headless-chromium-php#309]
Here https://github.com/chrome-php/chrome/blob/1.3/src/Dom/Node.php#L181 it looks like that internally the message 'setFileInputFiles' allows for an array of files -- but Node::sendfile() only supports one -- and then promotes it to an array for message.  

I would like support to allow multiple files

**Comment by djbusby** [comment]
I've patched my local version like below, and it's working great.

```
diff --git a/src/Dom/Node.php b/src/Dom/Node.php
index d284750..9af78a7 100644
--- a/src/Dom/Node.php
+++ b/src/Dom/Node.php
@@ -178,10 +178,19 @@ class Node
             ->typeText($text);
     }
 
-    public function sendFile(string $filePath): void
+    public function sendFile($filePath): void
     {
+        if (is_array($filePath)) {
+            // OK
+        } elseif (is_string($filePath)) {
+            // Promote
+            $filePath = [ $filePath ];
+        } else {
+            throw new \Exception('Invalid Parameter to sendfile()');
+        }
+
         $message = new Message('DOM.setFileInputFiles', [
-            'files' => [$filePath],
+            'files' => $filePath,
             'nodeId' => $this->nodeId,
         ]);
         $response = $this->page->getSession()->sendMessageSync($message);
```

**Comment by enricodias** [comment]
Modifying methods to add features like this isn't the best approach. We'll end up with a more complex method with 2 execution flows doing 2 things. We would also lose type for the argument.

Creating a new method for this feature will result in a cleaner code. I'll make a PR for that as soon as I can.

**Comment by enricodias** [comment]
Implemented in #317
